hugo-site/content/docs/introduction.md

```diff
@@ -70,3 +70,63 @@ Project Trident provides many features:
 -   **Easy system administration:** Project Trident provides many graphical tools for performing system administration.
 
 -   **Vibrant community:** Project Trident has a friendly and helpful community.
+
+### Security
+
+The Project Trident system is secure by default.
+This section is an overview of the built-in security features.
+There are also tips about increasing the security of the installed system beyond the configured defaults.
+
+The security features built into Project Trident include:
+
+-   **Naturally immune to viruses and other malware:** Most viruses are written to exploit the Windows operating system.
```

Comment:
-   **Naturally immune to viruses and other malware written for other Operating Systems:** Most viruses are written to exploit the Windows operating system.